src/main/java/org/testng/ReporterConfig.java

```diff
@@ -1,124 +1,10 @@
 package org.testng;
```

Comment:
@juherr - Why remove the contents of this java class? Couldn't we have just added a deprecation note and removed references to this class and used the new one?

Reply:
In fact, I don't know well how the class could be used outside of its internal usage.
I only found the usage of the method I keep in the class.

Even if we keep the rest of the content, I've no idea what will be the expected behavior.

I think it won't break anything and somewhere I'd prefer to be notified (and fix the rare case if any).

Thought?

Reply:
@juherr - What I meant was to just do the following:

1. Add a deprecation message to `org.testng.ReporterConfig`.
2. Remove reference through out the codebase to this class and have it changed to use `org.testng.internal.ReporterConfig`
3. In next release of testng we get rid of `ReporterConfig`.

The reason why I cited this was because, in your github search you have listed some code that is using this class. So I was thinking that perhaps we apply a proper deprecation strategy and avoid breaking those users who are using this class.

Reply:
Ok, done.
I do my best but I'm not sure it is 100% backward compatible due to the inner class Property.